app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,52 @@
+class CategoriesController < ApplicationController
+  def new
+    @category = Category.new
+  end
+
+  def index
+    @categories = Category.all
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params)
+    if @category.save
+      flash[:success] = "You created #{@category.title}"
+      redirect_to category_path(@category)
+    else
+      render :new
```

Comment:
be sure to handle errors. what happens if a user enters a blank category? ideally, we'd like for users to see a nice error message